cbdefense_connector.py

```diff
@@ -803,9 +820,10 @@ def handle_action(self, param):
 
 if __name__ == '__main__':
 
-    import pudb
     import argparse
 
+    import pudb
+
     pudb.set_trace()
 
     argparser = argparse.ArgumentParser()
```

Comment:
Add `verify` as a command-line as semgrep results will show a vulnerability at line 850 when this MR gets merged.

Also, replace `exit()` with `sys.exit()`

Reply:
@pdesai-crest, `exit()` at line #868 and # 885 should be `sys.exit()`